umbrella/router.go

```diff
@@ -58,12 +58,11 @@ type funcRoutingInfo struct {
 	Manual      string
 }
 
-// confirmFunctionIsRegistered is designed to eliminate case sensitivity when calling a registered function.
-// EqualFold() is used to determine that the requested function is equal to a registered function under Unicode case-folding.
+// confirmFunctionIsRegistered is designed to return function registry info regardless of case sensitivity.
 func confirmFunctionIsRegistered(reqFunc string, funcReg *map[string]funcRoutingInfo) (string, error) {
-	for key, _ := range funcReg {
+	for key := range funcReg {
 		if strings.EqualFold(reqFunc, key) {
-			return key, nil
+			return funcReg[key], nil
```

Comment:
I am pretty sure this will fail to return. The function type is string, and this is of type, *funcRoutingInfo.

Did you perform a `go build` on your code before submitting the review?

Reply:
Yikes, yeah -- I completely forgot about the return type.

`go build`: I haven't. Partially because I didn't think/know to, partially because this project isn't in my GOPATH, I didn't think I could. I'll give it a shot next time around.

Reply:
For posterity:
```
Bretts-MBP:umbrella bretthurst$ go build
router.go:16:2: cannot find package "github.com/aws/aws-lambda-go/events" in any of:
	/usr/local/go/src/github.com/aws/aws-lambda-go/events (from $GOROOT)
	/Users/bretthurst/Dev/go/src/github.com/aws/aws-lambda-go/events (from $GOPATH)
router.go:17:2: cannot find package "github.com/aws/aws-lambda-go/lambda" in any of:
	/usr/local/go/src/github.com/aws/aws-lambda-go/lambda (from $GOROOT)
	/Users/bretthurst/Dev/go/src/github.com/aws/aws-lambda-go/lambda (from $GOPATH)
router.go:18:2: cannot find package "github.com/phoenixcoder/slack-golang-sdk/auth" in any of:
	/usr/local/go/src/github.com/phoenixcoder/slack-golang-sdk/auth (from $GOROOT)
	/Users/bretthurst/Dev/go/src/github.com/phoenixcoder/slack-golang-sdk/auth (from $GOPATH)
```
So I did a `go get` on those three so I had them locally, and now build is actually returning errors in the code I can tackle.